transit/rest_api/forms/customer_type_master.py

```diff
@@ -24,6 +24,8 @@ class Meta:
 class CustomerTypeViewSet(BaseFormViewSet):
     filterset_class = CustomerFilter
     queryset = CustomerType.objects.all().order_by('-id')
+    filter_backends = [filters.SearchFilter]
+    search_fields = ['customer_type_name']
```

Comment:
Looking on 'legacy web app' filtering - I think that we need to consider filtering by 'id' too. 
search_fields = ['id', 'customer_type_name']

Reply:
Good catch, added.